djstripe/contrib/rest_framework/mixins.py

```diff
@@ -0,0 +1,44 @@
+from rest_framework.serializers import ModelSerializer
+from rest_framework.views import APIView
+
+from ...models import Customer
+from ...settings import STRIPE_LIVE_MODE, subscriber_request_callback
+
+
+class AutoCreateCustomerMixin(APIView):
```

Comment:
I agree with you. However, there is a calll to `super()`, that's the reason for inheriting from APIView. I don't know what would be more Pythonic: this mixin with Inheritance, or APIView subclasses (all of which would need to be subclassed, isn't it?). Let me know.

Reply:
If in doubt, copy what Django / DRF do :)

I don't think there's any problem with using super() in a mixin.  Equally though this could be a an APIView subclass, but I'd avoid subclassing in a mixin.